File: tests/py-files/test-do2.py
import sys

def main():

    x = 2
    while True:
    
        y = 1
        while True:
        
            print("Hello")
            y = y + 1
        
            if y < 3:
                continue
            else:
                break
        x = x * 2
    
        if x < 10:
            continue
        else:
            break
    return 0

if __name__ == '__main__':
    main()

File: tests/c-files/test-do2.c
#include <stdio.h>

int main()
{
    int x = 2;
    do
    {
        int y = 1;
        do
        {

            printf("Hello\n");

            y = y + 1;

        } while (y < 3);

        x = x * 2;

    } while (x < 10);

    return 0;
}
